src/api.v2/model/experiment.js

```diff
@@ -1,22 +1,91 @@
+const _ = require('lodash');
+
+/* eslint-disable func-names */
 const generateBasicModelFunctions = require('../helpers/generateBasicModelFunctions');
+const sqlClient = require('../../sql/sqlClient');
+const { aggregateIntoJson } = require('../../sql/helpers');
+
+const getLogger = require('../../utils/getLogger');
+
+const logger = getLogger('[ExperimentModel] - ');
 
-const tableName = 'experiment';
+const experimentTable = 'experiment';
+const experimentExecutionTable = 'experiment_execution';
 
-const selectableProps = [
+const experimentFields = [
   'id',
   'name',
   'description',
-  'processing_config',
+  'samples_order',
   'notify_by_email',
   'created_at',
   'updated_at',
 ];
 
 const basicModelFunctions = generateBasicModelFunctions({
-  tableName,
-  selectableProps,
+  tableName: experimentTable,
+  selectableProps: experimentFields,
 });
 
+const getExperimentData = async (experimentId) => {
+  const sql = sqlClient.get();
+
+  function query() {
+    this.select('*')
+      .from(experimentTable)
+      .leftJoin(experimentExecutionTable, `${experimentTable}.id`, `${experimentExecutionTable}.experiment_id`)
+      .where('id', experimentId)
+      .as('experiment_with_exec');
+  }
+
+  const experimentExecutionFields = [
+    'params_hash', 'state_machine_arn', 'execution_arn',
+  ];
+
+  const result = await aggregateIntoJson(query, experimentFields, experimentExecutionFields, 'pipeline_type', 'pipelines', sql);
+
+  return result[0];
+};
+
+const updateSamplePosition = async (experimentId, oldPosition, newPosition) => {
+  const sql = sqlClient.get();
+
+  // Sets samples_order as an array that has the sample in oldPosition moved to newPosition
+
+  const trx = await sql.transaction();
+
+  try {
+    const result = await trx(experimentTable).update({
+      samples_order: trx.raw(`(
+        SELECT jsonb_insert(samples_order - ${oldPosition}, '{${newPosition}}', samples_order -> ${oldPosition}, false)
```

Comment:
I think after this PR it'd be good to organize a short demo on basic stuff that we need to know for SQL. Like the `jsonb`, minimal migrations functionality, etc... I can help with it.

Reply:
in particular explaining how to run the migrations locally and manage the local infrastructure in general (migrating dynamo data into the db as well)